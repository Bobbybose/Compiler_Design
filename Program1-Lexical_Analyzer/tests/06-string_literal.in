"abc"
"here's a quotation mark: \""
"backslash: \\"