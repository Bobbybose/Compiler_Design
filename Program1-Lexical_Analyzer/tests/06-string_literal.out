string-literal: "abc": 1, 1
string-literal: "here's a quotation mark: \"": 2, 1
string-literal: "backslash: \\": 3, 1